js/ui/diagram/ui.diagram.commands.js

```diff
@@ -181,128 +181,56 @@ const DiagramCommands = {
     getOptions: function() {
         const { DiagramCommand } = getDiagram();
         return [
+            {
+                command: DiagramCommand.Units,
+                text: "Units",
+                widget: "dxSelectBox"
+            },
             {
                 command: DiagramCommand.PageSize,
                 text: "Page Size",
                 widget: "dxSelectBox",
                 getValue: (v) => {
-                    const vParts = v.split("|");
+                    const vParts = v.replace("[", "").replace("]", "").split(",");
```

Comment:
I'd suggest to use JSON.parse instead of manually parsing JSON.

Reply:
dxSelectBox doesn't work with objects (in core we use Size), so for items we use
var value = typeof item.value === "object" ? item.value.toString() : item.value;
And when a values is changed we return an object similar to Size 

                    const vParts = v.replace("[", "").replace("]", "").split(",");
                    return {
                        width: parseInt(vParts[0]),
                        height: parseInt(vParts[1])
                    };